ansible/roles/repo/tasks/redhat.yaml

```
@@ -37,15 +37,15 @@
     msg: "WARN: Your system is not using subscription manager fetching packages such as kernel-headers might fail"
   when:
     - ansible_distribution == 'RedHat'
-    - rhelorg != None or rheluser != None
+    - rhelorg.subscribed_pool is not defined or rheluser.subscribed_pool is not defined
```

I think we want to print this WARN only when both `rhelorg` **and** `rheluser` are not defined.
If 1 is set don't need to print it.

| rhelorg not defined  | rheluser not defined | WARN |
| ----------- | ----------- | ----------- |
| T      | T       | T | 
| T   | F        | F |
| F   | T        | F |
| F   | F        | F |

```suggestion
    - rhelorg.subscribed_pool is not defined and rheluser.subscribed_pool is not defined
```